project_iteration_sale_inheritance/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
```

Comment:
Codacy found an issue: [Statement seems to have no effect](https://app.codacy.com/gh/Numigi/odoo-project-addons/pullRequest?prid=6223602)